docs/pages/integrate.mdx

```diff
@@ -159,15 +159,85 @@ export default function EditorExample() {
 ```tsx
 "use client";
 
-import { renderers } from "@mod-protocol/react-ui-shadcn";
+import { ContextType, Embed } from "@mod-protocol/core";
+import {
+  contentMiniApps,
+  defaultContentMiniApp,
+} from "@mod-protocol/miniapp-registry";
 import { RenderEmbed } from "@mod-protocol/react";
-import { Embed } from "@mod-protocol/core";
+import { renderers } from "@mod-protocol/react-ui-shadcn/dist/renderers";
+import {
+  sendTransaction,
+  switchNetwork,
+  waitForTransaction,
+} from "@wagmi/core";
+import { useMemo } from "react";
+import { useAccount } from "wagmi";
+
+export function Embeds(props: { embeds: Array<Embed> }) {
+  const { address } = useAccount();
+
+  // Handle send transaction e.g. on click of a mint button
+  // This example uses wagmi
+  const onSendEthTransactionAction = useMemo(
+    () =>
+      async ({ data, chainId }, { onConfirmed, onError, onSubmitted }) => {
+        try {
+          const parsedChainId = parseInt(chainId);
+
+          // Switch chains if the user is not on the right one
+          await switchNetwork({ chainId: parsedChainId });
+
+          // Send the transaction
+          const { hash } = await sendTransaction({
+            ...data,
+            chainId: parsedChainId,
+          });
+          onSubmitted(hash);
+
+          // Wait for the transaction to be confirmed
+          const { status } = await waitForTransaction({
+            hash,
+            chainId: parsedChainId,
+          });
+
+          onConfirmed(hash, status === "success");
+        } catch (e) {
+          onError(e);
+        }
+      },
+    []
+  );
+
+  const context = useMemo<Omit<ContextType, "embed">>(() => {
+    return {
+      /* Required context */
+      api: process.env.NEXT_PUBLIC_API_URL,
+
+      /* Optional context */
+      user: {
+        id: "3",                // Current user's FID
```

Comment:
we should make fids Farcaster scoped or named so that mod's schema isn't coupled directly to Farcaster

Reply:
for example 
`farcaster.fid` may be better.

Reply:
this probably also applies to the nft creator metadata

Reply:
i think this part of the documentation was accidentally included as it's not yet implemented. i agree with the nft creator metadata point. will open an issue